scripts/workflow-template.yml

```diff
@@ -31,21 +31,19 @@ jobs:
 
     steps:
     - name: Setup Build Root
-      uses: allenevans/set-env@v2.0.0
-      with:
-        BUILD_ROOT: "[% build_root %]"
+      run: |
+        echo "BUILD_ROOT=[% build_root %]" >> $GITHUB_ENV
     - name: Prepare caching timestamp
-      id: cache_ts
       shell: cmake -P {0}
       run: |
         string(TIMESTAMP current_date "%Y-%m-%d-%H;%M;%S" UTC)
-        message("::set-output name=timestamp::${current_date}")
-    - uses: actions/cache@v1
+        file(APPEND GITHUB_ENV "timestamp=${current_date}")
```

Comment:
FIXME: this doesn't seem to work, maybe we can switch to using git SHA (or recreate this timestamp using bash).